__tests__/app.test.js

```diff
@@ -180,16 +180,47 @@ describe("8. GET /api/reviews", () => {
         );
       });
   });
-  it('should accept a category query', () => {
+  it("should accept a category query", () => {
     return request(app)
       .get("/api/reviews?category=owner")
       .expect(200)
       .then(({ body }) => {
         expect(body.length).toEqual(13);
-        expect(body[0].owner).toBe('mallionaire')
-          })
+        expect(body[0].owner).toBe("mallionaire");
+      });
   });
   it("status 404, should return an error", () => {
     return request(app).patch(`/api/renews`).expect(404);
   });
 });
+
+describe("9. GET /api/reviews/:review_id/comments", () => {
+  it("should ", () => {
+    return request(app)
+      .get("/api/reviews/2/comments")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.review.length).toEqual(3);
+        expect(
+          body.review.forEach((review) => {
+            expect(review).toEqual(
+              expect.objectContaining({
+                author: expect.any(String),
+                body: expect.any(String),
+                comment_id: expect.any(Number),
+                created_at: expect.any(String),
+                review_id: expect.any(Number),
+                votes: expect.any(Number),
+              })
+            );
+          })
+        );
+      });
+  });
+  it("status 404, should return an error", () => {
```

Comment:
ideally make these it strings a bit more descriptive; in what circumstances should the error be returned?